src/components/Trip.js

```diff
@@ -98,7 +106,9 @@ export default function Trip() {
             <div className="tripDetailsFieldContent">
               <i className="far fa-calendar-alt"></i>
               {singleTrip.checkoutDate.slice(8, 10)}.
-              {singleTrip.checkoutDate.slice(5, 7)}
+              {new Date(singleTrip.checkoutDate).toLocaleString("default", {
+                month: "short",
+              })}
```

Comment:
Just to leave a comment: you are using almost the same div with almost the same functions in it several times. You could make another component of it (I now, you are refactoring right now) . And give it the className of <i> and singleTrip key as a prop.

Reply:
Thanks Manuel